package.json

```diff
@@ -6,11 +6,16 @@
     "type": "git",
     "url": "git://github.com/mozilla/normandy.git"
   },
+  "bin": {
```

Comment:
I don't think you should need this at all? I might be wrong, but your lint stuff should still work without it.